DaisyMicheal/Ecommerce_web
# finalized landing page

**Pull Request Description Landing Page**

**What does this PR do?**
Create and style the e-commerce landing page for the project.

**Description of Task to be completed:**
- Design and implement the landing page layout, including sections for featured products, promotional banners, and testimonials.
- Integrate a responsive navigation bar with links to key pages.
- Ensure the landing page is fully responsive across different screen sizes.
- Add call-to-action buttons linking to product categories and the shop page.
- Implement smooth scrolling for anchor links within the page.
- Write unit tests for key components used in the landing page.
- Optimize images and assets for faster page load times.
- Add SEO-friendly meta tags and descriptions.

**How should this be manually tested?**
- Pull the branch and install dependencies if necessary.
- Navigate to the landing page and verify the layout across various screen sizes.
- Test the navigation bar links and anchor links for smooth scrolling.
- Check the responsiveness of the call-to-action buttons and promotional banners.
- Run the unit tests to ensure they pass.
- Verify the page’s SEO attributes using browser developer tools.

**Any background context you want to provide?**
None

**What are the relevant Pivotal Tracker stories?**
#188043034

**Screenshots (if appropriate):**
None

**Questions:**
None
